### PR TITLE
[1LP][RFR] Update set_conversion_host_api() to use CA certs

### PR DESCRIPTION
**Pull request by nachandr**
### PRT Run
{{ pytest: cfme/tests/v2v/test_v2v_smoke.py --use-template-cache --provider-limit 2 --use-provider rhv-ims --use-provider vsphere67-ims }}

PRT results:
510 - PASS
511 - PASS

### CODE REVIEW

File: cfme/fixtures/v2v_fixtures.py [hunk 1]
```diff
@@ -213,28 +213,35 @@ def vddk_url():
...
+            "cat /etc/pki/ovirt-engine/apache-ca.pem").output
+
         # Support for UCI hosts for RHV would be added in 5.11.5.
         if appliance.version >= '5.11.5':
             resource_type = "ManageIQ::Providers::Redhat::InfraManager::Vm"
             vm_key = conf.credentials[
                 target_provider.data["private-keys"]["engine-rsa"]["credentials"]]
             auth_user = vm_key.username
-            private_key = vm_key.password
+            # private_key = vm_key.password
```
Comment (sshveta) [inline cfme/fixtures/v2v_fixtures.py:234]: Remove comment

**Comment by mnadeem92** [comment]
@nachandr  as UCI is not coming with 5.11.5 release, can you update this PR only for VM refresh issue for upstream, we will enable UCI code only after UCI gets released.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7303efd53c9441cf6f409e608b831330a1db900c:

`cfme/fixtures/v2v_fixtures.py`:
- :bangbang: Line 224:9: **E112** *expected an indented block*
- :bangbang: Line 224:18: **E999** *IndentationError: expected an indented block*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/fixtures/v2v_fixtures.py (hunk 1 above)
Comment (nachandr) [reply cfme/fixtures/v2v_fixtures.py]: Done

**Pull request merged**